docs/source/developer/setup.rst

```diff
@@ -350,6 +354,6 @@ If you encounter any issue during installation, run:
 
 .. code-block:: shell
 
-    pip install -r requirements.txt -r requirements-test.txt
+    pip install . -r requirements-test.txt
```

Comment:
shouldn't this be:
```
pip install -e . -r requirements-test.txt
```

Does it work also without the `-e`?

Reply:
@pandafy can you reply to this please? :point_up:

Reply:
Earlier the instructions were to install the package using `python setup.py develop`. I have updated that to `pip install -e .[saml]. I have also updated this part too.